packages/Pill/src/Pill.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
+import pillStyles from "./Pill.styles";
+
+export const tokenPillColors = [
+  tokens.color.black,
+  tokens.color.blackLighten60,
+  tokens.color.blue,
+  tokens.color.greenDarken10,
+  tokens.color.orangeDarken10,
+];
+export const severityPillColors = ["noRisk", "lowRisk", "mediumRisk", "highRisk"];
+export const pillSizes = ["small", "medium"];
+
+const propTypes = {
+  a11yText: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  isDisabled: PropTypes.bool,
+  onClick: PropTypes.func,
+  pillColor: PropTypes.oneOf([...tokenPillColors, ...severityPillColors]),
+  size: PropTypes.oneOf(pillSizes),
+  textColor: PropTypes.string,
```

Comment:
I feel like we should just impose this, based on the `pillColor`.  Either programmatically, or with a hashmap, since we have a limited set of options we accept for `pillColor`.